schema_utils_test.py

```diff
@@ -68,6 +71,14 @@
 #     https://github.com/oppia/oppia/wiki/Schema-Based-Forms
 UI_CONFIG_SPECS = {
     SCHEMA_TYPE_BOOL: {},
+    SCHEMA_TYPE_CUSTOM: {
+        'hide_complex_extensions': {
```

Comment:
Why these changes? Probably needs documentation.

Also a bit concerned about whether this is the right place to put them, since they do not apply to all "custom"-type schemas. What is the design decision behind this and what alternatives did you consider?